src/Mailgun/Connection/RestClient.php

```diff
@@ -147,6 +147,9 @@ public function post($endpointUrl, array $postData = [], $files = [])
                     $postDataMultipart[] = [
                         'name' => sprintf('%s[%d]', $key, $index++),
                         'contents' => $subValue,
+                        'headers' => [
+                            'Content-Disposition' => sprintf('form-data; name="%s"', $key),
```

Comment:
We used this fix to use the `/v2/routes` endpoint (and deployed it succesfully in production). Are there any endpoints which do not support multiple resources with the same name?